apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs

```diff
@@ -122,4 +122,248 @@ private Operation PollForCompletion(Operation operation, string alias)
+                Instance instanceResource = new Instance
+                    {
+                        Name = instanceName,
+                        Zone = zone,
+                        Description = "тест",
+                        MachineType = machineType,
+                        Disks = {
+                            new AttachedDisk{
+                                AutoDelete = true,
+                                Boot = true,
+                                Type = AttachedDisk.Types.Type.Persistent,
+                                InitializeParams = init_params
+                            }
+                        },
+                        NetworkInterfaces = {
+                            new NetworkInterface{
+                                Name="default"
+                            }
+                        }
+                    };
+                Operation insertOp = instancesClient.Insert(projectId, zone, instanceResource);
+                PollForCompletion(insertOp, "create");
+            }
+        
+
+            void FetchInstance(){
+                _output.WriteLine($"Retrieving instance with the name: {instanceName}");
+                Instance instance = instancesClient.Get(projectId, zone, instanceName);
+                Assert.NotNull(instance);
+                Assert.Equal(instanceName, instance.Name);
+                Assert.Equal(machineType, instance.MachineType);
+                Assert.Equal(false, instance.ShieldedInstanceConfig.EnableSecureBoot);
+                Assert.Equal("тест", instance.Description);
+            }
+
+            void DeleteInstance(){
+                _output.WriteLine($"Deleting instance with the name: {instanceName}");
+                Operation deleteOp = instancesClient.Delete(projectId, zone, instanceName);
+                deleteOp = PollForCompletion(deleteOp, "delete");
+                _output.WriteLine($"Operation to delete instance completed: status {deleteOp.Status}; start time {deleteOp.StartTime}; end time {deleteOp.EndTime}");
+            }
+
+            void PatchInstance(){
+                ShieldedInstanceConfig shieldedInstanceConfigResource = new ShieldedInstanceConfig{
+                        EnableSecureBoot = true
+                    };
+                Operation stopOp = instancesClient.Stop(projectId, zone, instanceName);
+                PollForCompletion(stopOp, "stop");
+                Operation patchOp = instancesClient.UpdateShieldedInstanceConfig(
+                    projectId,
+                    zone,
+                    instanceName,
+                    shieldedInstanceConfigResource);
+                PollForCompletion(patchOp, "patch");
+                Instance instance = instancesClient.Get(projectId, zone, instanceName);
+                Assert.Equal(true, instance.ShieldedInstanceConfig.EnableSecureBoot);
+            }
+
+        }
+        [Fact (Skip="resize fails")]
+        public void TestQueryParams(){
```

Comment:
It's not clear why we'd need a second test for whether query parameters are transcoded properly or not.

Reply:
This is an edge case(a bug was found during testing for other client libraries) - we want to test that 0 won't be omitted in query parameters.